simbad/rotsearch/amore_search.py

```diff
@@ -183,6 +185,9 @@ def run(
         else:
             self.template_tmp_dir = os.path.join(default_tmp_dir, dir_name + "-{0}")
 
+        if not monitor:
+            monitor = self.percentage_complete
```

Comment:
Not sure what's going on with `monitor`, if it's intended to be `None` then this is fine but otherwise check that `self.percentage_complete` returns something :) Actually even if it's intended to stay as `None` then there's no point on assigning the output of `self.percentage_complete` on the first place...

Reply:
I'm just going to remove the monitor argument in run as it isn't currently being used.